src/github/queries.gql

```diff
@@ -160,6 +160,25 @@ fragment ReviewThread on PullRequestReviewThread {
 	}
 }
 
+fragment LegacyReviewThread on PullRequestReviewThread {
```

Comment:
I tried to use the `@skip` directive instead of having a whole new fragment, but it never worked. Commenting here in case someone sees this and has some idea why.